docs/userguides/cache.md

```diff
@@ -0,0 +1,73 @@
+# Cache
+
+Use the cache plugin to store provider data in an sqlite database.
+
+```bash
+ape cache init --network <your-choice-of-network>
+```
+
+You will need to set-up your network connection that you want to work off of.
+
+For example, if you want to work off of the infura provider, you have to install the infura plugin.
+
+```bash
+ape plugins install infura
+```
+
+Then, to connect to the mainnet:
+
+```bash
+ape cache init --network ethereum:mainnet:infura
```

Comment:
Shouldn't need the provider, I would just do `ethereum:mainnet` since any provider should be interchangeable for a given network

Reply:
This was implemented